src/Orchard.Web/Modules/Orchard.Email/Views/EditorTemplates/Parts/SmtpSettings.cshtml

```diff
@@ -65,6 +72,11 @@
             </span>
         </div>
     </div>
+    <div>
+        <label for="@Html.FieldIdFor(m => m.ListUnsubscribe)">@T("List-Unsubscribe header")</label>
```

Comment:
List-Unsubscribe is a STMP header that a mail client can understand natively.
https://www.postmastery.com/list-unsubscribe-header-critical-for-sustained-email-delivery/

It can help an email not go to junk.